exts/cesium.omniverse/mdl/cesium.mdl

```diff
@@ -11,64 +11,95 @@ module [[
+    gltf_texture_lookup_value base_color_texture = gltf_texture_lookup_value(),
+    uniform color base_color_factor = color(1.0),
+    uniform float metallic_factor = 1.0,
+    uniform float roughness_factor = 1.0,
+    uniform color emissive_factor = color(0.0),
+    uniform gltf_alpha_mode alpha_mode = opaque,
+    uniform float base_alpha = 1.0,
+    uniform float alpha_cutoff  = 0.5
+) [[ anno::hidden() ]] = let {
+    auto base_color_texture_value = base_color_texture.valid ? base_color_texture.value : float4(1.0);
+    auto final_base_color_texture_value = alpha_blend(imagery_layers_texture.value, base_color_texture_value);
+    auto final_base_color_texture = gltf_texture_lookup_value(true, final_base_color_texture_value);
+    auto final_base_color_factor = base_color_factor;
+
+    material base = gltf_material(
+        base_color_factor: final_base_color_factor,
+        base_color_texture: final_base_color_texture,
+        metallic_factor: metallic_factor,
+        roughness_factor: roughness_factor,
+        emissive_factor: emissive_factor,
+        alpha_mode: alpha_mode,
+        base_alpha: base_alpha,
+        alpha_cutoff: alpha_cutoff
     );
 
-    float3 final = float3(1.0, 1.0, 1.0);
+} in material(
+    thin_walled: base.thin_walled,
+    surface: base.surface,
+    volume: base.volume,
+    ior: base.ior,
+    geometry: base.geometry
+);
 
-    for (int i = 0; i < texture_count; i++) {
-        gltf_texture_lookup_value value = texture_values[i];
-        if (value.valid) {
-            float3 rgb = float3(value.value[0], value.value[1], value.value[2]);
-            float alpha = value.value[3];
-            final = alpha * rgb + (1.0 - alpha) * final;
+export gltf_texture_lookup_value cesium_imagery_layer_resolver(
+    uniform int imagery_layers_count = 0,
+    gltf_texture_lookup_value imagery_layer_0 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_1 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_2 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_3 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_4 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_5 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_6 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_7 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_8 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_9 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_10 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_11 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_12 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_13 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_14 = gltf_texture_lookup(),
+    gltf_texture_lookup_value imagery_layer_15 = gltf_texture_lookup()
```

Comment:
These should actually by `gltf_texture_lookup_value()`. It's harmless for now, but I fixed it on a different branch.

Reply:
Oh and I forgot to mention in the PR description, `eAsset` arrays don't seem to work in Fabric which is why each texture is its own entry instead of `gltf_texture_lookup_value[16]`